lib/app/views/index_onc_program.erb

```diff
@@ -202,11 +202,44 @@ For alternate installation options, please visit <a href='https://github.com/onc
   <div class="tab-pane" style="font-size: 20px;" id="pills-notes" role="tabpanel" aria-labelledby="pills-notes-tab">
     <div style="padding: 15px 0; font-size: 30px;">Release Notes</div>
 
+    <div style="padding: 0 0 15px 0; font-size: 24px;">Inferno Program Edition v1.8.0</div>
+    <p>
+      The Inferno Program Edition v1.8.0 release fixes a number of issues that may incorrectly cause systems to fail tests.
+      It also includes a minor fixes not directly related to test execution.
+      This release may result in some systems that previously passed to now fail due to additional MustSupport validations.
+    </p>
+    <p>
+      <i>Notice: Users that run Inferno Program Edition locally SHOULD rerun terminology build process, 
+      as 2021 version of some Code Systems are included in this release.
+      Please see <a href="https://github.com/onc-healthit/inferno-program/#terminology-support">the README</a> for instructions on how to
+      update your local copy of terminology files. Users MAY run this release without rebuilding terminology though tests may fail to 
+      validate codings from newer version of Code Systems.</i>
+    </p>    
+    <p>This version includes updates that:</p>
+    <ul>
+      <li>Adds MustSupport validation for Observation.component in FHIR Blood Pressue profile</li>
+      <li>Updates MustSupport validation for dataAbsentReason in Observation profiles that Systems that never provide an observation without a value are not required to support dataAbsentReason</li>
+      <li>Updates MustSupport validation on DocumentReference that server shall support at least one of content.attachment.url and content.attachment.data</li>
```

Comment:
```suggestion
      <li>Updates MustSupport validation on DocumentReference resources so that servers do not need to demonstrate support for both content.attachment.url and content.attachment.data</li>
```